src/plugins/chart_expressions/expression_metric/public/components/metric_vis.tsx

```diff
@@ -140,68 +146,127 @@ const getFormatter = (
     : new Intl.NumberFormat(locale, intlOptions).format;
 };
 
-const getColor = (value: number, paletteParams: CustomPaletteState | undefined) =>
-  paletteParams
-    ? getPaletteService().get('custom')?.getColorForValue?.(value, paletteParams, {
-        min: paletteParams.rangeMin,
-        max: paletteParams.rangeMax,
-      }) || defaultColor
-    : defaultColor;
+const getColor = (
+  value: number,
+  paletteParams: CustomPaletteState,
+  accessors: { metric: string; max?: string; breakdownBy?: string },
+  data: Datatable,
+  rowNumber: number
+) => {
+  let minBound = paletteParams.rangeMin;
```

Comment:
why do we assign `minBound` and `maxBound` to the value if then in line 160 we always reassign it?

Reply:
Leftovers from a previous version of this logic. Thanks for pointing this out! I will clean it up.